packages/tse-backend/src/modules/exchange-data/exchange-data.service.ts

```diff
@@ -0,0 +1,298 @@
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
+import { ExchangeRegistryService } from '../exchange-registry/exchange-registry.service';
+import { CustomLogger } from '../logger/logger.service';
+import {
+  GetMultipleTickerPricesCommand,
+  GetOHLCVCommand,
+  GetSupportedSymbolsCommand,
+  GetTickerPriceCommand,
+  GetTickersCommand,
+} from './model/exchange-data.model';
+import { ExchangeDataSubscriptionManager } from './subscription-manager.ws.service';
+import { createCompositeKey } from '../../common/utils/subscriptionKey';
+import { MarketDataType } from '../../common/enums/exchange-data.enums';
+import {
+  OHLCVResponse,
+  TickerPriceResponse,
+} from '../../common/interfaces/exchange-data.interfaces';
+
+@Injectable()
+export class ExchangeDataService {
+  private readonly logger = new CustomLogger(ExchangeDataService.name);
+
+  constructor(
+    private readonly exchangeRegistryService: ExchangeRegistryService,
+    @Inject(forwardRef(() => ExchangeDataSubscriptionManager))
+    private readonly subscriptionManager: ExchangeDataSubscriptionManager,
+  ) {}
+
+  async getTickers(command: GetTickersCommand) {
+    const { exchange, symbols } = command;
+    const exchangeInstance = this.exchangeRegistryService.getExchange(exchange);
+    if (!exchangeInstance || !exchangeInstance.has.fetchTickers) {
+      throw new Error(
+        'Exchange does not support fetchTickers or is not configured.',
+      );
+    }
+
+    const symbolsArray = symbols.split(',');
```

Comment:
Why symbols is a string instead of an list?

Reply:
Fixed: ce69c8d6